instances/latest/contentTypes/text_vnd.fsl.bvec.jsonld

```diff
@@ -0,0 +1,20 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/contentTypes/text_vnd.fsl.bvec",
+  "@type": "https://openminds.ebrains.eu/core/ContentType",
+  "dataType": null,
+  "description": "The FSL bvec format is used for ASCII text file that contains a list of gradient directions applied during diffusion weighted volumes. The order of entries in this file must match the order of volumes in the input data (4D NIFTI file).",
```

Comment:
It is essential for this type to specify that the file contains three rows: the first row for the x-axis, the second row for the y-axis, and the third row for the z-axis.